frontend/src/components/maps/leaflet/InfoSlideOut/AssociatedParcelsList.tsx

```diff
@@ -0,0 +1,51 @@
+import { IBuilding } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { Link, useLocation } from 'react-router-dom';
+import queryString from 'query-string';
+
+interface IAssociatedParcels {
+  /** the selected property information */
+  propertyInfo: IBuilding | null;
+}
+
+/**
+ * Component that displays the associated parcels of a building
+ * as a clickable list
+ * @param propertyInfo the selected building
+ */
+export const AssociatedParcelsList: React.FC<IAssociatedParcels> = ({ propertyInfo }) => {
```

Comment:
If I get some time I'd love to make a reusable component for this, we are doing this clickable list thing in a few places and I worry about consistency. This looks good though.